Source/Factories/StackClusterFactory.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  StackClusterFactory.swift
+//  Matrioska
+//
+//  Created by Mathias Aichinger on 11/01/2017.
+//  Copyright © 2017 runtastic. All rights reserved.
+//
+
+import Foundation
+
+extension ClusterFactory {
+    
+    /// A concrete ComponentFactory which produces a stack cluster component
+    public final class StackClusterFactory: ComponentFactory {
+        public func produce(children: [Component],
```

Comment:
it seems to me that these factories have something wrong.
To comform to ClusterFactory you need a function that given children and meta creates a `Component`
then `ClusterLayout.stack...` `ClusterLayout.tab...` are already factories so we are creating all this code to basically wrap those functions.

Other than this this factory implementation seems wrong to me because it's only working for `.cluster`
How do you create wrapper and view from json? Am I missing anything?